src/sentry/models/authprovider.py

```diff
@@ -162,7 +152,7 @@ def disable_scim(self, user):
             # Only one SCIM installation allowed per organization. So we can reset the idp flags for the orgs
             # We run this update before the app is uninstalled to avoid ending up in a situation where there are
             # members locked out because we failed to drop the IDP flag
-            self._reset_idp_flags()
+            organization_service.reset_idp_flags(organization_id=self.organization_id)
```

Comment:
Interesting. The code surrounding this operation is also all control silo models. (SentryAppInstallationForProvider, SentryAppInstall, and SentryApp). Perhaps we should move this entire block into the new RPC method?

Reply:
@markstory I don't think we can do that. 🤔  `organization_service` is a region silo service.